component/tests/golden/vshn/appcat/appcat/21_composition_vshn_postgres.yaml

```diff
@@ -1001,22 +1003,21 @@ spec:
                         for: 1h
                         labels:
                           severity: warning
-                  - name: postgresql-memory
-                    rules:
```

Comment:
This looks broken, `name` is now deeper and the `rules` field is missing.

Reply:
Fixed